meetings/2023-04-11-Community-Meeting.md

```diff
@@ -0,0 +1,94 @@
+# Gluster Community Meeting - 11/04/2023
+
+### Previous Meeting minutes:
+
+- [Meeting Minutes](https://github.com/gluster/community/tree/master/meetings)
+
+### Date/Time: Check the [community calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDQ0YmRydTllMXYzdWFoMmpsbjdqNXJlYmNfMjAyMDEwMjdUMDkwMDAwWiBzYWptb2hhbUByZWRoYXQuY29t&tmsrc=sajmoham%40redhat.com&scp=ALL)
+
+### Bridge
+
+- Monthly, 2nd Tuesday, 02:30PM IST
+- Bridge: https://meet.google.com/cpu-eiue-hvk
+
+---
+
+### Attendance
+
+Name (#gluster-dev alias) - company
+
+- Niraj Yadav - IBM
+- Sanju Rakonde - PhonePe
+- Sunil Kumar Acharya - IBM
+- Shwetha Acharya - IBM
+
+### Community
+
+- Project metrics:
+
+| Metrics                                                                                                            | Value (Last meeting) |
+| ------------------------------------------------------------------------------------------------------------------ | -------------------- |
+| [Coverity](https://scan.coverity.com/projects/gluster-glusterfs)                                                   | 61 (88)              |
+| [Clang Scan](https://build.gluster.org/job/clang-scan/lastBuild/)                                                  | 64 (64)              |
+| [Test coverage](https://build.gluster.org/job/line-coverage/lastCompletedBuild/Line_20Coverage_20Report/)          | 71.3 (71.2)          |
+| [Gluster User Queries in last 14 days](https://lists.gluster.org/pipermail/gluster-users/2023-April/thread.html) | 2                    |
+| [Total Github issues](https://github.com/gluster/glusterfs/issues)                                                 | 183                 |
+
+- Any release updates?
+
+  - [Shwetha] - Release 10.4 can be expected by end of this week. (Impacted due to easter vacations)
+
+- Blocker issues across the project?
+
+  - None
+
+- Notable thread from mailing list / User stories
+  - None
+
+### Conferences / Meetups
+
+- None
+
+### GlusterFS - v11.0 and beyond
```

Comment:
11.0 was already released, no?

Reply:
Yes (14th Feb). This is just a placeholder in the MoM for any suggestions related to future updates.